src/etc/rc.subr.d/livemode

```diff
@@ -58,12 +58,12 @@ if (is_install_media()) {
 
         echo "\n";
 
-        $greeting  = "Welcome!  Both `root' and `installer' users are available for system\n";
-        $greeting .= "setup or invoking the installer, respectively.  The predefined root\n";
-        $greeting .= "password works for both accounts.";
+        $greeting  = "Welcome!  OPNsense is running in \"live\" mode from install media.\n" .
+                     "Please login as 'root' to continue in live mode, or as 'installer' to install OPNsense.\n" .
+                     "Use the default or previously-imported root password for both accounts.";
 
         if (!isset($config['system']['ssh']['noauto']) && is_process_running('sshd')) {
-            $greeting .= "  Remote login via SSH is possible.";
+            $greeting .= "  Remote login via SSH is also enabled.";
```

Comment:
Hmm... kind of 70/30 ish :) On the one hand, any wording removable without loss of meaning - simplify. On the other hand, it's standard English for another option after describing a first, to emphasise the alternative: "You can login THIS way. THAT way is ALSO okay." For me the 2nd of these wins, but not by a lot :) Do what you think best.

Ande going forward, please do modify and commit if it's something trivial, if it'll save a trip round waiting for me and then re-reviewing. (Eg the "live" quote marks, code format fixes, minor typo). Courtesy appreciated however!

Reply:
I'll have to assume that if you submit it you want it so we need to discuss :) it's a difficult area, even though it seems trivial. Words matter, otherwise easily overlooked or misunderstood and the clarification is certainly welcome. Let's try this and see if it can be squeezed into 3 lines of 80 characters. :)